.github/workflows/python.yml

```diff
@@ -0,0 +1,36 @@
+name: Python bindings
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: build and test python bindings using setuptools
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+
+    - name: install dependencies (mac)
+      if: runner.os == 'macOS'
+      run: |
+        brew install boost-build boost boost-python3 python@3.9
+
+    - name: install dependencies (linux)
+      if: runner.os == 'Linux'
+      run: |
+        sudo apt install libboost-tools-dev libboost-python-dev libboost-dev libboost-system-dev python3 python3-setuptools
```

Comment:
you should probably rebase now that https://github.com/arvidn/libtorrent/pull/5749 landed, as well as add those `sudo apt update` here as well